add assembly steps for sealion cubesat

cscha010 opened the pull request:
closes #188

capsulecorplab committed:
add boilerplate assemblySteps for sealion-cubesat

kchiu002 committed:
Add assembly steps an parts for sealion-structure

capsulecorplab committed:
replace liquid with jinja2 template for pdf-theme

capsulecorplab committed:
replace liquid with jinja2 template for pdf-theme

capsulecorplab committed:
update build to use local doctools if installed

capsulecorplab committed:
update build to use local doctools if installed

cscha010 committed:
Merge branch 'main' into system-model/add-assembly-steps-for-sealion-cubesat/188

cscha010 committed:
Updated parts.yaml for sealion.structure with plates and brackets for the outer structure

capsulecorplab committed:
update build to generate assembly instructions

capsulecorplab committed:
update build to generate assembly instructions

capsulecorplab committed:
update dof-helpers submodule to d216f2b

cscha010 committed:
Added SeaLion Structure tools

cscha010 committed:
Add package.json for bottom plate, monopole option

cscha010 committed:
Added package.json for bottom plate, SBand Patch option.

cscha010 committed:
Updated the sealion-structure parts.yaml file to meet the current component list of the sealion cubesat

cscha010 committed:
Changed bottom-plate-monopole and bottom-plate-sband package.json files to accomodate the "sealion" prefix and updated the structure's parts.yaml to accurately display the proper fields.

capsulecorplab committed:
fix issue with $workdir env var not resetting

capsulecorplab committed:
fix podman build for manuscript-example.pdf

capsulecorplab committed:
fix install for m30mlTools/node_modules if missing

capsulecorplab reviewed (changes requested):
added some notes for changes

cscha010 committed:
Updated the parts.yaml file to hyphonated names.

Created tool folders with package.json files for each part's various options.

Added authors and licensing to each packagel.json

Changed parts.yaml descriptions to human-readable names and moved detailed descriptions to notes.

capsulecorplab committed:
add comments as to what buildDocs.sh will generate

capsulecorplab committed:
build assembly instructions as HTML from asciidoc

capsulecorplab committed:
add symlink to dof-helpers in sealion-cubesat

capsulecorplab committed:
add symlink to dof-helpers in sealion-structure

capsulecorplab committed:
build assembly instructions as pdf in buildDocs.sh

capsulecorplab committed:
add generateAssemblyInstructions.sh

capsulecorplab committed:
add symlink to generateAssemblyInstructions.sh in sealion-cubesat

capsulecorplab committed:
add symlink to generateAssemblyInstructions.sh in sealion-structure

capsulecorplab committed:
Merge branch 'main' into system-model/add-assembly-steps-for-sealion-cubesat/188

cscha010 committed:
Added sheetmetal components

Trouble shooting the assemblySteps.yaml steps

Commented out all parts.yaml and tools.yaml code that was not relevant to the assemblySteps.yaml query

cscha010 committed:
Added precautions.yaml file

Allowed successful generation of assembly instructions

cscha010 committed:
Added sheetmetal components and their options

capsulecorplab committed:
fix assemby instructions build via podman

for running in project root directory

capsulecorplab committed:
add missing tools.yaml to sealion-cubesat

capsulecorplab committed:
add missing parts.yaml to project root

cscha010 committed:
Created sealion-board-assembly

Added parts.yaml and package.json in sealion-board-assembly

Moved communication, obc, and power folders into sealion-board-assembly/components

Created sealion-interface-module into sealion-board-assembly/components

Created uhf antenna, magnetometer, and battery pack into sealion-structure/components

capsulecorplab committed:
update expected node version to v18.5.0 in build

capsulecorplab committed:
use node package for asciidoctor

capsulecorplab committed:
create variables for setting path to node binaries

capsulecorplab committed:
remove asciidoctor from package.json

capsulecorplab committed:
use locally installed asciidoctor

capsulecorplab committed:
add dof-cubesat (linkml schema) as submodule

capsulecorplab committed:
create symlinks for dof-cubesat in subcomponents

Commit:
Adding both General Photovoltaic Array Structures plus Orbital STK Source
